packages/web3/src/components/balances-summary.tsx

```diff
@@ -59,7 +59,7 @@ export function BalancesSummary() {
           return (
             <div
               key={id}
-              className="text-foreground flex min-w-0 items-center gap-3 px-2 text-sm font-medium last:pb-2"
+              className="text-foreground flex items-center gap-3 px-4 py-2 text-sm font-medium"
```

Comment:
nit but let's do little less vertical padding, IMO it's too much now
```suggestion
              className="text-foreground flex items-center gap-3 px-4 py-1 text-sm font-medium"
```

Reply:
Okay, done.